python/cudf/cudf/core/groupby/legacy_groupby.py

```diff
@@ -79,7 +79,7 @@ def group_min(data, segments, output):
 
 
 class Groupby(object):
-    """Groupby object returned by cudf.DataFrame.groupby().
+    """Groupby object returned by cudf.DataFrame.groupby(method="cudf").
```

Comment:
I think we should say something about the implications of `method="cudf"`.

My initial inclination was to suggest 'Nulls are unsupported with `method="cudf"`', but it appears it's more complicated than that. Aggregation functions will _run_ against Series having null masks, but the result will be garbage.

For example:
```
import cudf

df = cudf.DataFrame()
df['id'] = [0, 1, 2, 2, 3]
df['val'] = [0, 1, 2, None, 3]

# correct result
df.groupby(by=['id']).mean()

# incorrect for id=2 and id=3
df.groupby(by=['id'], method='cudf').mean()
```
Incorrect result:
```

  | id | val
-- | -- | --
0 | 0.0
1 | 1.0
2 | 2.5
3 | 0.0
```

Reply:
Have updated the doctstring for the legacy groupby class.